src/pages/index.tsx

```diff
@@ -1,8 +1,33 @@
+import Link from "next/link";
+
+const HOME_INDEX = [
+  {
+    id: 1,
+    title: "룰렛",
+    href: "/roulette",
+  },
+  {
+    id: 2,
+    title: "더치페이",
+    href: "/dutch-pay",
+  },
+  {
+    id: 3,
+    title: "만날 위치",
+    href: "/where-to-meet",
+  },
+];
+
 export default function Home() {
   return (
-    <>
-      <h1 className="text-2xl font-bold mb-4">환영합니다!</h1>
-      <p>원하시는 기능을 선택해주세요.</p>
-    </>
+    <div className="flex flex-col gap-28 py-14 px-10">
+      {HOME_INDEX.map((domain) => {
+        return (
+          <Link key={domain.id} href={domain.href}>
+            <span className="text-4xl font-semibold">{domain.title}</span>
+          </Link>
+        );
+      })}
```

Comment:
```suggestion
      {HOME_INDEX.map((domain) => (
          <Link key={domain.id} href={domain.href}>
            <span className="text-4xl font-semibold">{domain.title}</span>
          </Link>
          )}
```

요로코롬 써도 좋을 것 같아요~!

Reply:
그리고 구조 분해 할당을 비선호 하시는 편 일까요? 구조 분해 할당을 해도 깔끔해 보일 것 같은데요~!

```suggestion
      {HOME_INDEX.map(({id, href, title}) => {
        return (
          <Link key={id} href={href}>
            <span className="text-4xl font-semibold">{title}</span>
          </Link>
        );
      })}
```

Reply:
27f5bd551a1096df8dfd64574efe4240a2f6c9e3
오오! 조금 더 깔끔해진 것 같아요!